### PR TITLE
Conv net train

### PR DESCRIPTION
**Pull request by pozo123**
Still needs to pull the images from directory and classify them for the generator + test-train split

### CODE REVIEW

File: src/train/train_conv_net.py [hunk 1]
```diff
@@ -1,6 +1,65 @@
...
-    pass
+    #HERE Get data and split it in diferent directories for the train generator
+    #HERE Split into train and test to verify that the code works
+
+    #HERE verify addressess and params
+    weights_address = 'backend/weights'
+    images_address_tr = 'train/images_tr'
+    images_address_tr = 'train/images_ts'
+    num_classes = 5
+    batch_size = 1000
```
Comment (sms1097) [inline src/train/train_conv_net.py]: Batch size should be power of 2, see [this](https://datascience.stackexchange.com/questions/20179/what-is-the-advantage-of-keeping-batch-size-a-power-of-2)

File: src/train/train_conv_net.py [hunk 2]
```diff
@@ -1,6 +1,65 @@
...
+    model.compile(optimizer='adam', loss='sparse_crossentropy', metrics=['acc'])
+
+    history = model.fit(
+        train_generator,
+        epochs=epochs
+        )
+
+    model.evaluate(test_generator)
+
+    model.load_weights(weights_address)
```
Comment (sms1097) [inline src/train/train_conv_net.py]: save weights

**Review by sms1097 — CHANGES_REQUESTED** [review]
Also run flake8 on this

File: src/train/train_conv_net.py (hunk 1 above)
Comment (sms1097) [reply src/train/train_conv_net.py]: Also, slightly picky, but `num_classes` and `batch_size` should probably be final variable (declared at top all caps)

File: src/train/train_conv_net.py [hunk 3]
```diff
@@ -1,6 +1,65 @@
...
+
```
Comment (sms1097) [inline src/train/train_conv_net.py:73]: Also add if to run file (`if __name__ == '__main__'`)

**Review by sms1097 — CHANGES_REQUESTED** [review]

File: src/train/data_splitter.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                cat3.append(datum)
+            elif categories[3] in datum:
+                cat4.append(datum)
+            elif categories[4] in datum:
+                cat5.append(datum)
+                
+        return cat1, cat2, cat3, cat4, cat5
+    
+    def fill_directories(data, training, testing, split_size):
+        train = random.sample(data, round(split_size * len(data)))
```
Comment (sms1097) [inline src/train/data_splitter.py:44]: Lets just seed random in this file `random.seed(1)`

**Review by sms1097 — CHANGES_REQUESTED** [review]
Also we should remove cat and dog references! 🐱

**Comment by sms1097** [comment]
@pozo123 This commit is too large, can you remove the unnecessary files before we review?

**Comment by sms1097** [comment]
@pozo123 Can you seed random and remove `.DS_store` files?

File: src/train/data_splitter.py (hunk 4 above)
Comment (pozo123) [reply src/train/data_splitter.py:44]: Just saw this comment haha I did remove all cat and dog references, these "cats" stand for "categories"!

**Review by sms1097 — APPROVED** [review]

**Review by aaa1008 — APPROVED** [review]
